backend/graminsta/core/serializers.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: UTF-8 -*-
+
+"""
+Here defines model serializers.
+"""
+
+from django.contrib.auth import models as auth_models
+from rest_framework import serializers
+from .models import UserInfo
+
+
+class UserSerializer(serializers.ModelSerializer):
+    """
+    UserSerializer defined for User model.
```

Comment:
This documentation does not give extra information. Try to give something useful.

e.g. `Serializer that serializes/deserializes User object